mono/metadata/w32process-win32-uwp.c

```diff
@@ -71,11 +71,14 @@ ves_icall_System_Diagnostics_Process_ShellExecuteEx_internal (MonoW32ProcessStar
 MonoStringHandle
 ves_icall_System_Diagnostics_Process_ProcessName_internal (HANDLE process, MonoError *error)
 {
-	gunichar2 name [MAX_PATH]; // FIXME MAX_PATH
-	const guint32 len = GetModuleFileNameW (NULL, name, G_N_ELEMENTS (name));
-	if (len == 0)
+	gunichar2* name = NULL;
```

Comment:
```suggestion
	gunichar2 *name = NULL;
```

Reply:
Take stars out of types where they make more sense.